_218622274409:
  extrinsics:
    rotation:
    - - -0.0021086243643426705
      - -0.9999853380699821
      - 0.004987719754932951
    - - -0.029126885726194096
      - -0.004924197617343751
      - -0.999563593177402
    - - 0.9995734981636836
      - -0.0022529808896200043
      - -0.029116075386763196
    translation:
    - - 3.1975433616047866
    - - 154.7300333833169
    - - 74.03220750000061
  intrinsics:
    camera_matrix:
    - - 433.93536376953125
      - 0.0
      - 427.9855041503906
    - - 0.0
      - 433.37115478515625
      - 244.14837646484375
    - - 0.0
      - 0.0
      - 1.0
    dist_coeffs:
    - - -0.053600188344717026
      - 0.06146441772580147
      - -0.0007368515944108367
      - 0.0005973638035356998
      - -0.020658744499087334
_826212070364:
  extrinsics:
    rotation:
    - - -0.5141936617023972
      - 0.8082188767284055
      - -0.2870315759022954
    - - 0.7799114710242443
      - 0.3013693528394217
      - -0.5485568434847455
    - - -0.3568514755887667
      - -0.5059236705956963
      - -0.785301384120197
    translation:
    - - 504.1024539533356
    - - 309.40147626640874
    - - 381.46323511627895
  intrinsics:
    camera_matrix:
    - - 617.4249267578125
      - 0.0
      - 324.92291259765625
    - - 0.0
      - 617.6627197265625
      - 246.35433959960938
    - - 0.0
      - 0.0
      - 1.0
    dist_coeffs:
    - - 0.0
      - 0.0
      - 0.0
      - 0.0
      - 0.0
_941322072865:
  extrinsics:
    rotation:
    - - 0.3404181528780481
      - 0.8463128877181177
      - -0.4097193884518015
    - - 0.8488289353164334
      - -0.08915590632491233
      - 0.5210956370350392
    - - 0.4044810499401272
      - -0.5251720865103209
      - -0.7487251563756412
    translation:
    - - 506.7169744501197
    - - -342.40746169095337
    - - 313.39697265855256
  intrinsics:
    camera_matrix:
    - - 611.9188842773438
      - 0.0
      - 323.4823913574219
    - - 0.0
      - 612.2329711914062
      - 232.13735961914062
    - - 0.0
      - 0.0
      - 1.0
    dist_coeffs:
    - - 0.0
      - 0.0
      - 0.0
      - 0.0
      - 0.0
